docs/DeveloperGuide.md

```diff
@@ -249,14 +249,29 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a…                    | I can…                                                   | So that I can…                                                                                          |
+| -------- | ------------------------ | -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| * *      | Meticulous OHS admin     | have automatic backups of my data                        | rest knowing my data will not be accidentally erased.                                                   |
```

Comment:
[These issues](https://github.com/AY2021S1-CS2103-T16-3/tp/issues?page=2&q=is%3Aissue+is%3Aopen+label%3Atype.Story) are missing.

Reply:
Also missing are #27, #25, #24, #23.

Reply:
#27 is already in. I'll include the rest.

Reply:
Perhaps you forgot to include #58?